src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1238,9 +1238,8 @@
 					marqueeDiv.classList.add("ui-marquee");
 				}
 				widget = ns.widget.Marquee(marqueeDiv, {
-					marqueeStyle: "scroll",
-					ellipsisEffect: "gradient",
-					iteration: "infinite",
+					marqueeStyle: "endToEnd",
+					iteration: 1,
```

Comment:
ellipsisEffect: "gradient" is not required for entToEnd?

Reply:
I'll fix it

Reply:
Gradient is default value and is not needed in this case.